fetch.bs

```diff
@@ -3335,6 +3340,9 @@ optionally with a <i>recursive flag</i>, run these steps:
    <li><p><a href=https://w3c.github.io/webappsec-upgrade-insecure-requests/#upgrade-request>Upgrade <var>request</var> to a potentially secure URL, if appropriate</a>.
    [[!UPGRADE]]
 
+   <li><p><a abstract-op lt="set the Fetch metadata headers for a request">Set the Fetch metadata headers for <var>request</var></a>.
```

Comment:
Which is this an abstract-op? I thought those were only for JavaScript.

Reply:
Bikeshed defines `abstract-op` as "[English-language algorithms](https://tabatkins.github.io/bikeshed/#dfn-types)"; that's generally why I've been using it to identify algorithms. What type would you use?

Reply:
Interesting, I tend to use dfn for those and only use abstract-op for the IAmACamelCaseAlgorithm types from ECMAScript. I guess I'm not going to block on this then, but copying @domenic and @tabatkins for thoughts.

Reply:
I think it's appropriate to use abstract-op here, fwiw.

Reply:
(When I finally add some cool functionality around registering algorithms with arguments and types and whatnot, I expect that I'll hook abstract-op as the type for that.)